api/query/cache/service/main.go

```diff
@@ -205,11 +219,12 @@ func searchHandler(w http.ResponseWriter, r *http.Request) {
 		var netClient = &http.Client{
 			Timeout: time.Second * 5,
 		}
-		resp.MetadataResponse = shared.GetMetadataResponse(runs, netClient, log.StandardLogger())
+		resp.MetadataResponse = shared.GetMetadataResponse(runs, netClient, logrus.StandardLogger())
```

Comment:
We are inside a request context here, right? Shall we pass in `log` instead?

Reply:
```suggestion
		resp.MetadataResponse = shared.GetMetadataResponse(runs, netClient, log)
```